### PR TITLE
Update Project section layout to side-by-side

### PR DESCRIPTION
**Pull request by sdotpeng**
1. I used flex to position project element side by side. 
2. Changed the id to class, as per #2.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/FormHandlerServlet.java [hunk 1]
```diff
@@ -13,16 +13,22 @@ public class FormHandlerServlet extends HttpServlet {
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+    String firstName = request.getParameter("first-name");
```
Comment (IslandCow) [inline portfolio/src/main/java/com/google/sps/FormHandlerServlet.java]: Are these variables used? I can't tell why you're reading them from the request.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -9,82 +9,122 @@
...
+        <div class="card introduction-card">
             <img src="images/profile.png" alt="Avatar" style="width:100%">
             <div class="container">
                 <h4><b>Siyuan (Ricky) Peng</b></h4>
                 <p>
-                    <a href='https://github.com/sdotpeng/'>GitHub<a/>  | 
-                    <a href='https://www.linkedin.com/in/siyuanrickypeng/'>LinkedIn<a/>  |  
-                    <a href='https://leetcode.com/sdotpeng/'>LeetCode<a/>
+                    <a href='https://github.com/sdotpeng/'>GitHub<a /> |
+                        <a href='https://www.linkedin.com/in/siyuanrickypeng/'>LinkedIn<a /> |
```
Comment (IslandCow) [inline portfolio/src/main/webapp/index.html:32]: indentation.  If you right click in this doc in the editor, it'll format things for you. It's called "Format Document" or "Format Selection"

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -9,82 +9,122 @@
...
+                <label>Enter Your Email: 
+                    <input type="email" name="email" required />
+                </label>
+            </fieldset>
+            <fieldset>
+                <label><input type="radio" name="form-type" id="recruiter" class="inline" /> I'm a recruiter</label>
+                <label><input type="radio" name="form-type" id="feedback" class="inline" /> Feedback for portfolio</label>
+            </fieldset>
+            <fieldset>
+                <label>Leave a message::
```
Comment (IslandCow) [inline portfolio/src/main/webapp/index.html:122]: Two colons

**Comment by sdotpeng** [comment]
Added form to the HTML. It seems that I cannot make multiple PRs on this branch. @IslandCow

**Review by IslandCow — APPROVED** [review]
Looks good.  I look forward to seeing the request parameters used.

**Pull request merged**